src/lib/list/selection-list.spec.ts

```diff
@@ -0,0 +1,277 @@
+    fixture.detectChanges();
+    let listOption = fixture.debugElement.query(By.directive(MdListOption));
+    let listItemEl = fixture.debugElement.query(By.css('.mat-list-item'));
+
+    expect(listItemEl.nativeElement.classList).not.toContain('mat-list-item-focus');
+
+    listOption.nativeElement.focus();
+    fixture.detectChanges();
+    expect(listItemEl.nativeElement.classList).toContain('mat-list-item-focus');
+
+    listOption.nativeElement.blur();
+    fixture.detectChanges();
+    expect(listItemEl.nativeElement.classList).not.toContain('mat-list-item-focus');
+  });
+
+  it('should be able to dispatch selected one item', () => {
+    let fixture = TestBed.createComponent(SelectionListWithListOptions);
+    fixture.detectChanges();
+    let listOption = fixture.debugElement.queryAll(By.directive(MdListOption));
+    let selectionList = fixture.debugElement.query(By.directive(MdSelectionList));
+    let testListItem = listOption[2].injector.get<MdListOption>(MdListOption);
+    let selectList = selectionList.injector.get<MdSelectionList>(MdSelectionList).selectedOptions;
+
+    expect(selectList.selected.length).toBe(0);
+
+    testListItem.toggle();
+    fixture.detectChanges();
+
+    expect(selectList.selected.length).toBe(1);
+  });
+
+  it('should be able to dispatch selected more than one items', () => {
+    let fixture = TestBed.createComponent(SelectionListWithListOptions);
+    fixture.detectChanges();
+    let listOption = fixture.debugElement.queryAll(By.directive(MdListOption));
+    let selectionList = fixture.debugElement.query(By.directive(MdSelectionList));
+    let testListItem = listOption[2].injector.get<MdListOption>(MdListOption);
+    let testListItem2 = listOption[1].injector.get<MdListOption>(MdListOption);
+    let selectList = selectionList.injector.get<MdSelectionList>(MdSelectionList).selectedOptions;
+
+    expect(selectList.selected.length).toBe(0);
+
+    testListItem.toggle();
+    fixture.detectChanges();
+
+    testListItem2.toggle();
+    fixture.detectChanges();
+
+    expect(selectList.selected.length).toBe(2);
+  });
+
+  it('test deselect an option', () => {
+    let fixture = TestBed.createComponent(SelectionListWithListOptions);
+    fixture.detectChanges();
+    let listOption = fixture.debugElement.queryAll(By.directive(MdListOption));
+    let selectionList = fixture.debugElement.query(By.directive(MdSelectionList));
+    let testListItem = listOption[2].injector.get<MdListOption>(MdListOption);
+    let selectList = selectionList.injector.get<MdSelectionList>(MdSelectionList).selectedOptions;
+
+    expect(selectList.selected.length).toBe(0);
+
+    testListItem.toggle();
+    fixture.detectChanges();
+
+    expect(selectList.selected.length).toBe(1);
+
+    testListItem.toggle();
+    fixture.detectChanges();
+
+    expect(selectList.selected.length).toBe(0);
+  });
+
+  it('test disabled items cannot be selected', () => {
```

Comment:
Nit: if you can, the test names should start with 'should', so it reads like a sentence:

`it('should not select disabled items')`